internal/batches/errors.go

```diff
@@ -77,3 +77,9 @@ func (e IgnoredRepoSet) Append(repo *graphql.Repository) {
 func (e IgnoredRepoSet) HasIgnored() bool {
 	return len(e) > 0
 }
+
+type ValidationError struct{ Reason string }
+
+func (e ValidationError) Error() string {
+	return fmt.Sprintf("validation failed: %s", e.Reason)
+}
```

Comment:
I think I want to have this to distinguish validation things from actual errors, so we can in the future have better error reporting. Can remove it, though.